components/HackathonModal.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
can you just add a comment to the top of the file explaininng what this is. I only ask because this is a hardcoded temporary component rather then a composable or configurable component like everything else.

in fact should this be a layout? because its so specific.